doc/source/metadata/Probabilities.rst

```diff
@@ -69,12 +65,14 @@ which runs every hour to generate a 3-member ensemble:
 
 
 Percentiles
-***********
+-----------
 
 This is probably the second most strightforward form,
-as again it still represents actual sets of values of the diagnostic,
-but instead of a set of realizations (consistent over time),
-the set of percentile values represent the values at a set of probability levels.
+as again it still represents actual sets of values of the diagnostic.
+Instead of a set of realizations (consistent over time),
+the set of percentile values represent the values of a set of thresholds
```

Comment:
```suggestion
the set of percentiles represent thresholds
```
This is currently quite word-y, and the words "values" and "set" appear a few times each, which makes it more difficult to read